Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) Coconut Lizard Limited. All rights reserved.
+
+#include "ImPlotWidgets/Public/LineFilledGraphWidget.h"
+#include "ImGui/Public/ImPlotImplementation.h"
+
+void SFilledGraphWidget::Construct(const FArguments& InArgs)
+{
+	InData = InArgs._InDataArray;
+
+	ChildSlot
+		[
+			SAssignNew(ImGuiCustomWidget, SImGuiCustomWidget)
+			.ContextIndex(InArgs._ContextIndex)
+			.OnImGuiDrawDelegate(this, &SFilledGraphWidget::OnImGuiDraw)
+		];
+}
+
+void SFilledGraphWidget::OnImGuiDraw() const
+{
+	constexpr int32 WindowFlags = ImGuiWindowFlags_NoDecoration
+		| ImGuiWindowFlags_NoBackground
+		| ImGuiWindowFlags_NoSavedSettings;
+
+	constexpr int32 GraphFlags = ImPlotFlags_CanvasOnly | ImPlotFlags_AntiAliased;
+
+	constexpr int32 AxisFlags = ImPlotAxisFlags_NoDecorations;
+
+	const FGeometry& AllottedGeometry = ImGuiCustomWidget.Get()->GetTickSpaceGeometry();
+
+	ImGui::SetNextWindowPos(ImVec2(0, 0), ImGuiCond_Always);
+	ImGui::SetNextWindowSize(ImVec2(AllottedGeometry.GetLocalSize().X, AllottedGeometry.GetLocalSize().Y), ImGuiCond_Always);
+	ImGui::Begin("ImPlot Demo", NULL, WindowFlags);
```

Comment:
NULL is bad and should be replaced with nullptr. _Unless_ the API you're calling specifically expects NULL.

Reply:
NULL is just a macro defined as 0, this can be implicitly cast to a null pointer but can also be interpreted as a literal zero

Reply:
I thought this was for whether the imgui widget has the option to be closed so I changed it to false, but all I could really find on it is that it can be closed when this value is set to != NULL. So I'm not really sure what would be the best solution?

Reply:
My guess would be that this is the situation Josef specified where the API specifically expects NULL. If this is this case my preference would be to define the variable before so that we know what it does at first glance. So whatever the parameter is called in the ImGui::Begin function, call it that. Maybe something like `constexpr uint32 CanBeClosed = NULL;`

Reply:
Well, the _type_ of that definition must match the type of the ImGui::Begin param, but otherwise yes that.